datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -270,7 +304,10 @@ def download(self, target_relpath):
             If download over TUF and in-toto is successful, this function will
             return the complete filepath to the desired target.
         """
-        target_abspath = self.__download_with_tuf_in_toto(target_relpath)
+        if self.__disable_verification:
+            target_abspath = self.__download_without_tuf(target_relpath)
```

Comment:
It might be a good idea to log a warning so users are aware of turned off verification even in logs.

Reply:
Done in e48da10